sdk/storage/storage-file-share/src/generated/src/models/index.ts

```diff
@@ -1756,6 +1971,10 @@ export interface DirectoryForceCloseHandlesHeaders {
    * Contains count of number of handles closed.
    */
   numberOfHandlesClosed?: number;
+  /**
+   * Contains count of number of handles that failed to closed.
+   */
+  numberOfHandlesFailedToClosed?: number;
```

Comment:
FYI: lots of changes not related to this PR might be auto involved during generating protocol layer code, please be aware of this, and ensure functionality correctness before releasing.

Reply:
Yeah, will have a thorough review of the generated code when all features are done for this release.